src/exercise/01.md

```diff
@@ -68,4 +68,4 @@ using JavaScript as well.
 ## 🦉 Feedback
 
 Fill out
-[the feedback form](https://ws.kcd.im/?ws=React%20Fundamentals%20%E2%9A%9B&e=01%3A%20Basic%20JavaScript-rendered%20Hello%20World&em=).
+[the feedback form](https://ws.kcd.im/?ws=React%20Fundamentals%20%E2%9A%9B&e=01%3A%20Basic%20JavaScript-rendered%20Hello%20World&em=grant%40alley.co).
```

Comment:
might as well commit the email changes to your repo in a separate commit so they don't surface in every PR as a diff